config.toml

```diff
@@ -1,16 +1,69 @@
 # The URL the site will be built for
 base_url = "https://ardusub.com"
+title = "ArduSub"
```

Comment:
Since this commit is changing a file added initially on the same patch, I would recommend to squash this commit with the first one (7a61f687324591853fbfb824451979469e1ee09c).
That can accomplished with a "git commit fixup" or via "git rebase interactive", I can help you with both